packages/react-components/src/templates/NetworkTeam.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { TeamResponse } from '../../../model/src';
```

Comment:
I'm a little bit scared that this works :fearful: ... can you replace those imports with `@asap-hub/model` everywhere (at least one more occurrence in this PR) please?

Reply:
Ah, VSCode auto import fail. I'll keep an eye out for that